src/CTA.Rules.Models/ProjectConfiguration.cs

```diff
@@ -10,13 +10,13 @@ public class ProjectConfiguration
         public ProjectConfiguration()
         {
             TargetVersions = new List<string> { Constants.DefaultCoreVersion };
-            PackageReferences = new Dictionary<string, string>();
+            PackageReferences = new Dictionary<string, Tuple<string, string>>();
```

Comment:
Is there a reason for using `Tuple<string, string>` instead of an object? My thought is that since the original and new versions are always coupled, using an object to contain them could improve readability.